mmv1/templates/terraform/custom_delete/kms_crypto_key.erb

```diff
@@ -8,9 +8,9 @@
 and all its CryptoKeyVersions will be destroyed, but it will still be present in the project.`, cryptoKeyId.cryptoKeyId())
 
 	// Delete all versions of the key
-	if err := clearCryptoKeyVersions(cryptoKeyId, userAgent, config); err != nil {
-		return err
-	}
+	//if err := clearCryptoKeyVersions(cryptoKeyId, userAgent, config); err != nil {
```

Comment:
The current failures are all in CryptoKey tests (not CryptoKeyVersion) - the errors are:

```

    provider_test.go:315: Step 4/4 error: Check failed: Check 2/3 error: CryptoKey tf-test-tz16saqy9g should have no versions, but version projects/tf-test-818832964805778275/locations/us-central1/keyRings/tf-test-k89y9yowbf/cryptoKeys/tf-test-tz16saqy9g/cryptoKeyVersions/1 has state ENABLED
```

This failure is likely due to these lines being commented out. Could you uncomment them? If that causes another error it would be great to see in CI.

Reply:
Thanks for pointing out. I just tried locally and I got **"message": "The request cannot be fulfilled. Resource projects/tf-test-1111180836049777411/locations/us-central1/keyRings/tf-test-071bb6qyfe/cryptoKeys/tf-test-9gmcgjtnnf/cryptoKeyVersions/1 has value DESTROY_SCHEDULED in field crypto_key_version.state."**

I'm a bit confused why this is happening since we do have a code block to handle error if there's any. (I did add a line with HandleNotFoundError).

If I remove the HandleNotFoundError and put 
if err != nil {
	return nil
}
at kms.utils.go Line ~230, the test will pass, but it's a bit unsafe to handle error case like the above.

Reply:
I think it works now